machinery/topology.go

```diff
@@ -186,6 +190,141 @@ func (t *Topology) ToDot() string {
 	return t.graph.String()
 }
 
+func (t *Topology) Graph() *dot.Graph {
+	return t.graph
+}
+
+// noisy keys to remove from top-level
+var noisyKeys = map[string]struct{}{
```

Comment:
we could keep this stuff out of the lib - in our case, filter up in the operator

Reply:
Yes, I'm good with a quick marshal for now and have the consumers filter

Reply:
Perfect - think that makes sense. thanks @KevFan